services/web/server/src/simcore_service_webserver/projects/exceptions.py

```diff
@@ -217,3 +217,7 @@ class ClustersKeeperNotAvailableError(BaseProjectError):
 
 class InvalidInputValue(WebServerBaseError):
     msg_template = "Invalid value for input '{node_id}': {message} for value={value}"
+
+
+class ProjectGroupNotFoundError(BaseProjectError):
+    msg_template = "Project group not found. {reason}"

```

Comment:
THOUGHT: I advocate for having an exception per error type instead of adding parameters as `reason` or `message`. 
This way we keep all "human readable errors" in one place and e.g. it is easier to review, localize etc

Reply:
I am just consistent with how we do it in that module. I do not like that suddenly we start to do something differently. If we agree we are doing it from now on in a new way, let's create an issue and split the load between us to change it everywhere based on new standards.

Reply:
OK, did you create the issue. You can do it from the `...` button on the top right of the comment

Reply:
Nice, I didn't know that. Ok, I have created one.